### PR TITLE
feat: the preparation for subgraph deployment

### PR DESCRIPTION
**Pull request by DevZi1la**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,13 +4,14 @@
   "scripts": {
     "codegen:rinkeby": "graph codegen -- subgraph-rinkeby.yaml",
     "build:rinkeby": "graph build -- subgraph-rinkeby.yaml",
-    "deploy:rinkeby": "graph deploy --node https://api.thegraph.com/deploy/ --ipfs https://api.thegraph.com/ipfs/ -- vbstreetz/witswap-staking-rinkeby subgraph-rinkeby.yaml",
+    "deploy:rinkeby": "graph deploy --node https://api.thegraph.com/deploy/ --ipfs https://api.thegraph.com/ipfs/ -- joyfulmagician/taraswap_staker-rinkeby subgraph-rinkeby.yaml",
```
Comment (DevZi1la) [inline package.json]: We don't need any of these rinkeby deployments etc

Comment (DevZi1la) [reply package.json]: Please delete

File: package.json [hunk 2]
```diff
@@ -4,13 +4,14 @@
...
     "codegen:mainnet": "graph codegen -- subgraph-mainnet.yaml",
     "build:mainnet": "graph build -- subgraph-mainnet.yaml",
-    "deploy:mainnet": "graph deploy --node https://api.thegraph.com/deploy/ --ipfs https://api.thegraph.com/ipfs/ -- vbstreetz/witswap-staking subgraph-mainnet.yaml"
+    "deploy:mainnet": "graph deploy --node https://api.thegraph.com/deploy/ --ipfs https://api.thegraph.com/ipfs/ -- joyfulmagician/taraswap_staker subgraph-mainnet.yaml",
+    "deploy:local": "graph deploy --node http://localhost:8020/ --ipfs http://localhost:5001 taraswap_staker subgraph-mainnet.yaml"
```
Comment (DevZi1la) [inline package.json]: besides this we need a `create-local` command and a `remove-local` one. For reference please look into the `v3-subgraph` repo

File: package.json (hunk 1 above)
Comment (joyfulmagician) [reply package.json:7]: "codegen:mainnet": "graph codegen -- subgraph-mainnet.yaml",
"build:mainnet": "graph build -- subgraph-mainnet.yaml",
"deploy:mainnet": "graph deploy --node https://api.thegraph.com/deploy/ --ipfs https://api.thegraph.com/ipfs/ -- joyfulmagician/taraswap_staker subgraph-mainnet.yaml",
"deploy:local": "graph deploy --node http://localhost:8020/ --ipfs http://localhost:5001 taraswap_staker subgraph-mainnet.yaml",

File: package.json (hunk 2 above)
Comment (joyfulmagician) [reply package.json:11]: "create:local": "graph create --node http://localhost:8020/ taraswap_staker",
"remove:local": "graph remove --node http://localhost:8020/ taraswap_staker"

File: package.json [hunk 3]
```diff
@@ -2,15 +2,15 @@
   "name": "uniswap-v3-staker-subgraph",
   "license": "MIT",
   "scripts": {
-    "codegen:rinkeby": "graph codegen -- subgraph-rinkeby.yaml",
-    "build:rinkeby": "graph build -- subgraph-rinkeby.yaml",
-    "deploy:rinkeby": "graph deploy --node https://api.thegraph.com/deploy/ --ipfs https://api.thegraph.com/ipfs/ -- vbstreetz/witswap-staking-rinkeby subgraph-rinkeby.yaml",
     "codegen:mainnet": "graph codegen -- subgraph-mainnet.yaml",
     "build:mainnet": "graph build -- subgraph-mainnet.yaml",
-    "deploy:mainnet": "graph deploy --node https://api.thegraph.com/deploy/ --ipfs https://api.thegraph.com/ipfs/ -- vbstreetz/witswap-staking subgraph-mainnet.yaml"
+    "deploy:mainnet": "graph deploy --node https://api.thegraph.com/deploy/ --ipfs https://api.thegraph.com/ipfs/ -- joyfulmagician/taraswap_staker subgraph-mainnet.yaml",
```
Comment (DevZi1la) [inline package.json]:
```suggestion

```

File: subgraph-rinkeby.yaml [hunk 4]
```diff
@@ -11,7 +11,7 @@ dataSources:
       startBlock: 8528046
```
Comment (DevZi1la) [inline subgraph-rinkeby.yaml]: This file we don't need

**Pull request merged**